Mu2eG4/geom/TSdA_v02.txt

```diff
@@ -10,7 +10,8 @@ double tsda.halfLength4  = 25.4;
 double tsda.z0           = 4204.6; // z coordinate of center in Mu2e
 string tsda.materialName = "Polyethylene094";
 
-int    tsda.version        = 2; //increment version to no longer override z0 position
+int    tsda.version      = 2;  //increment version to no longer override z0 position
+int    tsda.build        = 1;  // P.M. : may want to set to false
```

Comment:
From the code I expected to see different versions of parameters for different 'builds'. How are different parameters then implemented? And why is 'build' needed as a new dimension instead of just incrementing the version?